README.md

```diff
@@ -94,98 +93,116 @@ Commands:
 
 AWS Options:
-  --client-request-token  a unique, case-sensitive string of up to 64 ASCII characters used to ensure idempotent retries.
-  --region                AWS region. Can also be set via --environment & stack-args.yaml:Region.
-  --profile               AWS profile. Can also be set via --environment & stack-args.yaml:Profile. Use --profile=no-profile to override values in stack-args.yaml and use AWS_* env vars.
-  --assume-role-arn       AWS role. Can also be set via --environment & stack-args.yaml:AssumeRoleArn. This is mutually exclusive with --profile. Use --assume-role-arn=no-role to override values in stack-args.yaml and use AWS_* env vars.
+  --client-request-token  a unique, case-sensitive string of up to 64 ASCII characters used to ensure idempotent retries.         [string] [default: null]
+  --region                AWS region. Can also be set via --environment & stack-args.yaml:Region.                                 [string] [default: null]
+  --profile               AWS profile. Can also be set via --environment & stack-args.yaml:Profile. Use --profile=no-profile to override values in
+                          stack-args.yaml and use AWS_* env vars.                                                                 [string] [default: null]
+  --assume-role-arn       AWS role. Can also be set via --environment & stack-args.yaml:AssumeRoleArn. This is mutually exclusive with --profile. Use
+                          --assume-role-arn=no-role to override values in stack-args.yaml and use AWS_* env vars.                 [string] [default: null]
 
 Options:
-  --environment, -e  used to load environment based settings: AWS Profile, Region, etc.
-  --debug            log debug information to stderr.
-  -v, --version      show version information
-  -h, --help         show help
+  --environment, -e  used to load environment based settings: AWS Profile, Region, etc.                                  [string] [default: "development"]
+  --debug            log debug information to stderr.                                                                           [boolean] [default: false]
+  --log-full-error   log full error information to stderr.                                                                      [boolean] [default: false]
+  -v, --version      show version information                                                                                                    [boolean]
+  -h, --help         show help                                                                                                                   [boolean]
 
 Status Codes:
   Success (0)       Command successfully completed
   Error (1)         An error was encountered while executing command
   Cancelled (130)   User responded 'No' to iidy prompt or interrupt (CTRL-C) was received
 ```
 
-### The `argsfile` (aka `stack-args.yaml`)
+### Environment Variables
```

Comment:
I think we should discourage people from wrapping iidy in complex shell scripts that use these envvars. For that reason, I'd move this section done.